jib-core/src/main/java/com/google/cloud/tools/jib/image/json/ContainerConfigurationTemplate.java

```diff
@@ -20,8 +20,8 @@
 import com.google.cloud.tools.jib.image.DescriptorDigest;
 import com.google.cloud.tools.jib.json.JsonTemplate;
 import com.google.common.annotations.VisibleForTesting;
-import java.util.ArrayList;
-import java.util.List;
+import com.google.common.collect.ImmutableSortedMap;
+import java.util.*;
```

Comment:
I don't know why IntelliJ keeps doing this

Reply:
I think there's a configuration you have to disable.

Reply:
https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports